Gruntfile.js

```diff
@@ -11,10 +11,21 @@
 var modRewrite = require('connect-modrewrite');
 var serveStatic = require('serve-static');
 
+var capitalize = (str) => {
+  return str.charAt(0).toUpperCase() + str.substr(1);
+};
+
 module.exports = function (grunt) {
   var contextRoot = grunt.option('contextRoot') || "dev-console";
   var isMac = /^darwin/.test(process.platform) || grunt.option('mac');
-
+  var specifiedUnitTestBrowsers = grunt.option('browsers') &&
+                                  grunt.option('browsers').split(',').map(capitalize);
+  // Travis & Jenkins expect Chrome
+  var defaultUnitTestBrowsers = isMac ? ['Firefox', 'Chrome'] : ['Chrome'];
+  var unitTestBrowsers = specifiedUnitTestBrowsers ?
+                          specifiedUnitTestBrowsers :
+                          defaultUnitTestBrowsers;
+  var e2eTestBrower = grunt.option('browser') || 'chrome';
```

Comment:
Typo: "Brower". Should be `integrationTestBrowser` anyway.

I'd rather not call it e2e since there's an e2e suite that's just part of the integration tests.

Reply:
Thx, will update. I remove the old `e2e.js` and replaced it with some things in `/test/integration/features/` that cover the same flow.  That said, I can still call this `integration` if its preferable.

Reply:
Looks like this typo still exists. @benjaminapetersen Did you push an update?

Reply:
thx, @spadgett, got it!